src/java.base/share/classes/jdk/internal/classfile/impl/ClassPrinterImpl.java

```diff
@@ -929,9 +947,9 @@ private static Node[] attributesToTree(List<Attribute<?>> attributes, Verbosity
                     nodes.add(map("enclosing method",
                             "class", ema.enclosingClass().name().stringValue(),
                             "method name", ema.enclosingMethodName()
-                                    .map(Utf8Entry::stringValue).orElse("null"),
+                                    .map((Function<Utf8Entry, ConstantDesc>)Utf8Entry::stringValue).orElse(BSM_NULL_CONSTANT),
```

Comment:
```suggestion
                                    .<ConstantDesc>map(Utf8Entry::stringValue).orElse(BSM_NULL_CONSTANT),
```
same for other casts.